modules/pgamsspBidAdapter.md

```diff
@@ -0,0 +1,79 @@
+# Overview
+
+```
+Module Name: PGAMSSP Bidder Adapter
+Module Type: PGAMSSP Bidder Adapter
+Maintainer: info@pgammedia.com
+```
+
+# Description
+
+Connects to PGAMSSP exchange for bids.
+PGAMSSP bid adapter supports Banner, Video (instream and outstream) and Native.
+
+# Test Parameters
+```
+    var adUnits = [
+                // Will return static test banner
+                {
+                    code: 'adunit1',
+                    mediaTypes: {
+                        banner: {
+                            sizes: [ [300, 250], [320, 50] ],
+                        }
+                    },
+                    bids: [
+                        {
+                            bidder: 'pgamssp',
+                            params: {
+                                placementId: 'testBanner',
```

Comment:
We'll need to see each example in your adapter's md file to return a valid bid response